core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -419,48 +419,6 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
       scheduler.workerRemoved(workerId, host, message)
     }
 
-    /**
-     * Mark a given executor as decommissioned and stop making resource offers for it.
-     */
-    private def decommissionExecutor(executorId: String): Boolean = {
```

Comment:
@holdenk ... to answer your question: This block of code was moved to the base class `ExecutorAllocationClient`. So the code in `ExecutorAllocationClient` is not "New". Furthermore, the semantics of this code were changed as it was moved to now unconditionally replenish the executors.

Reply:
Makes sense.